backend/src/schema/resolvers/follow.spec.js

```diff
@@ -87,32 +129,39 @@ describe('follow', () => {
     })
   })
   describe('unfollow user', () => {
+    beforeEach(async () => {
+      await mutate({ mutation: mutationFollowUser(user2.id) })
```

Comment:
You can either setup the data through a mutation or you can do:

```js
await myUser.relateTo(anotherUser, 'following')
```

If `myUser` and `anotherUser` is a neode node (that's the return type of the factory method).

Reply:
@roschaefer I wouldn't agree on this. Suggested improvement manipulates DB "directly" rather than through known server API, but we test API and we shouldn't reason about the DB or any other tools used by the server.

In the future relation name, `neode` model or some other properties can change which will break the test setup. So we'll have to get back and fix the test.

If we keep the current approach the test setup will break only if we change mutation definition. However, if this happens we'll have to rewrite all the test anyways.